judgels-frontends/raphael/src/routes/uriel/contests/ContestsPage/ContestsPage.tsx

```diff
@@ -55,21 +106,52 @@ class ContestsPage extends React.Component<ContestsPageProps, ContestsPageState>
     }
 
     const { data: contests, rolesMap } = response;
+    if (!contests || !rolesMap) {
```

Comment:
```
const contests = response.data!;
const rolesMap = response.rolesMap!;
```

Reply:
response.data can be null of it is loading